auslib/web/public/client.py

```diff
@@ -29,6 +29,33 @@ def getHeaderArchitecture(buildTarget, ua):
         return 'Intel'
 
 
+def getSystemCapabilities(systemCapabilities):
```

Comment:
Can the name of this function be "getInstructionSet", since it extract the instruction set from systemCapabilities query string?